README.md

```
@@ -31,249 +31,785 @@
 ```
 
-Ou, caso já tenha compilado o `.jar`:
+---
+
+## Infraestrutura e Deploy
+
+### Arquitetura de Serviços
 
-```bash
-java -jar target/*.jar
+```
```

markdownlint-cli2 (0.22.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)